src/api/tests/feature_tests/features/createCpmProduct.failure.feature

```diff
@@ -17,8 +17,9 @@ Feature: Create CPM Product - failure scenarios
       | partOf.identifier.system | https://directory.spineservices.nhs.uk/ORD/2-0-0/organisations |
       | partOf.identifier.value  | F5H1R                                                          |
     When I make a "POST" request with "default" headers to "ProductTeam/f9518c12-6c83-4544-97db-d9dd1d64da97/Product" with body:
-      | path | value            |
-      | name | My Great Product |
+      | path            | value                                |
+      | name            | My Great Product                     |
+      | product_team_id | f9518c12-6c83-4544-97db-d9dd1d64da97 |
```

Comment:
i don't think this is needed is it?

Reply:
Ah no, I must've added them to the wrong file, this one shouldn't be changed at all 🤦‍♀️

Reply:
product_team_id and product_name are both required params for the createCpmProduct

Reply:
only the product name is, `product_team_id` is inferred through the path as of the last release

Reply:
product_team_id is part of the path, shouldn't need to come thorugh in the body